### ISSUE TITLE
Multiselect is not accessible in Windows High Contrast Mode

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#6888]
## Environment

> Operating system

Windows 10 latest

> Browser

Edge, Firefox

> Assistive technology used to verify

Windows High Contrast Mode

## Detailed description
Multiselect in inaccessible on both Edge and Firefox
- missing focus states
- missing highlight state
- missing overall focus on input

Firefox
![image](https://user-images.githubusercontent.com/40970507/93941952-0fc55880-fcf5-11ea-9fdd-7f90bc2ca608.png)

Edge
![image](https://user-images.githubusercontent.com/40970507/93941967-18b62a00-fcf5-11ea-8441-525504159cac.png)
